httpclient5/src/main/java/org/apache/hc/client5/http/impl/io/PoolingHttpClientConnectionManager.java

```diff
@@ -408,8 +408,9 @@ public void release(final ConnectionEndpoint endpoint, final Object state, final
                     LOG.debug("{} connection {} can be kept alive {}", ConnPoolSupport.getId(endpoint), ConnPoolSupport.getId(conn), s);
                 }
             } else {
-                if (LOG.isDebugEnabled()) {
-                    LOG.debug("{} connection is not kept alive", ConnPoolSupport.getId(endpoint));
+                if (this.log.isDebugEnabled()) {
```

Comment:
@nfawcett Where does `this.log` come from? I do not see  `log` instance variable anywhere.